using-mobile-extensions/adobe-target/README.md

```diff
@@ -47,16 +47,54 @@ To add Target to your app:
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
-1. Add the Mobile Core and Target CocoaPods to your project via your `Podfile`.
+{% tab title="iOS (AEP 3.x)" %}
+1. Add the AEPCore, AEPIdentity and AEPTarget CocoaPods to your project via your `Podfile`.
 
    ```text
-    pod 'ACPCore'
-    pod 'ACPTarget'
+    pod 'AEPCore','~>3.0.0'
+    pod 'AEPIdentity','~>3.0.0'
+    pod 'AEPTarget','~>3.0.0'
    ```
 
 2. Import the Target and Identity libraries.
 
+  **Swift**
+
+   ```swift
+       import AEPCore
+       import AEPTarget
+       import AEPIdentity
+   ```
+
+   **Objective C**
+
+   ```objectivec
+       @import AEPCore
+       @import AEPTarget
+       @import AEPIdentity
+   ```
+   
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+1. Add the ACPCore, ACPIdentity and ACPTarget CocoaPods to your project via your `Podfile`.
+
+   ```text
+    pod 'ACPCore','~>2.0.0'
+    pod 'ACPIdentity','~>2.0.0'
+    pod 'ACPTarget','~>2.0.0'
```

Comment:
Change it to `~> 2.0` (implies up to but not including version 3.0)

Reply:
You do not need `ACPIdentity` here as it is bundled with ACPCore. Fix the description on L80 as well to only include ACPCore and ACPTarget.